internal/backends/selector/Cargo.toml

```diff
@@ -21,8 +21,7 @@ eventloop-winit-wayland = ["i-slint-backend-winit/wayland"]
 
 renderer-femtovg = ["i-slint-backend-winit/renderer-femtovg"]
 
-rtti-winit = ["i-slint-backend-winit/rtti"]
-rtti-qt = ["i-slint-backend-qt/rtti"]
+rtti = ["i-slint-backend-winit?/rtti", "i-slint-backend-qt?/rtti"]
```

Comment:
i guess also  `i-slint-core/rtti`
it is not needed because the interpreter also sets it, but it might still be good to be explicit.

Reply:
I take that back. It's not needed here, but it's needed in winit/qt (otherwise the macro generated code doesn't compile), and it is there.